### PR TITLE
ECHR and RS db setup routines merge.

### PR DESCRIPTION
**Pull request by Cloud956**
Merging latest (hopefully final) changes to the backend.

### CODE REVIEW

File: .env
Comment (shashankmc) [inline .env:1]: Can you provide a comment on the purpose of those starting dates please? How is this different from providing it in Airflow?

**Review by shashankmc — CHANGES_REQUESTED** [review]
Can we move towards utilizing proper logging methods rather than print statements please? Or would you rather fix that in a separate issue/branch?

File: .env
Comment (Cloud956) [reply .env:1]: I included comments for these variables in .env in the newest update.

**Review by shashankmc — APPROVED** [review]
Great, thanks for the changes!

**Pull request merged**